src/components/Calculator.js

```diff
@@ -1,74 +1,101 @@
+      </header>
+
+      <div className="calc-grid">
+        <section className="calc-numbers">
+          <button id="btn-ac" className="button_calculator" type="button" onClick={() => handleClick('AC')}>
+            AC
+          </button>
+          <button id="btn-minus-plus" className="button_calculator" type="button" onClick={() => handleClick('+/-')}>
+            +/-
+          </button>
+          <button id="btn-percentage" className="button_calculator" type="button" onClick={() => handleClick('%')}>
+            %
+          </button>
+          <button id="btn-one" className="button_calculator" type="button" onClick={() => handleClick('1')}>
+            1
+          </button>
+          <button id="btn-two" className="button_calculator" type="button" onClick={() => handleClick('2')}>
+            2
+          </button>
+          <button id="btn-three" className="button_calculator" type="button" onClick={() => handleClick('3')}>
+            3
+          </button>
+          <button id="btn-four" className="button_calculator" type="button" onClick={() => handleClick('4')}>
+            4
+          </button>
+          <button id="btn-five" className="button_calculator" type="button" onClick={() => handleClick('5')}>
+            5
+          </button>
+          <button id="btn-six" className="button_calculator" type="button" onClick={() => handleClick('6')}>
+            6
+          </button>
+          <button id="btn-seven" className="button_calculator" type="button" onClick={() => handleClick('7')}>
+            7
+          </button>
+          <button id="btn-eight" className="button_calculator" type="button" onClick={() => handleClick('8')}>
+            8
+          </button>
+          <button id="btn-nine" className="button_calculator" type="button" onClick={() => handleClick('9')}>
+            9
+          </button>
+          <button id="btn-zero" className="button_calculator" type="button" onClick={() => handleClick('0')}>
+            0
+          </button>
+          <button id="btn-period" className="button_calculator" type="button" onClick={() => handleClick('.')}>
+            .
+          </button>
+        </section>
+
+        <section className="calc-operators">
+          <button id="btn-divide" className="button_calculator" type="button" onClick={() => handleClick('/')}>
+            /
+          </button>
+          <button id="btn-multiply" className="button_calculator" type="button" onClick={() => handleClick('x')}>
+            *
+          </button>
+          <button id="btn-subtract" className="button_calculator" type="button" onClick={() => handleClick('-')}>
+            -
+          </button>
+          <button id="btn-add" className="button_calculator" type="button" onClick={() => handleClick('+')}>
+            +
+          </button>
+          <button id="btn-equals" className="button_calculator" type="button" onClick={() => handleEquals()}>
+            =
+          </button>
```

Comment:
- [ ] According to the requirements, you should be passing props from the `Calculator` component to a child component. Kindly create a child component and pass props to it. I would advise you to create a `Button` component and pass the properties of buttons as props to it. 👍 😎